### PR TITLE
Tanya's content and Ron's feedback

### PR DESCRIPTION
**Pull request by merriemcgaw**
Incorporating Ron's feedback. More (many) PRs to come

### CODE REVIEW

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- [AccessibleEvents](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.accessibleevents?view=netframework-4.7) indicating StateChange and NameChange when drop down is expanded or collapsed to `T:System.Windows.Forms.ToolStripDropDownItem`
+- The `T:System.Windows.Forms.ToolStripDropDownButton` control property has a ControlType property value of `F:System.Windows.Automation.ControlType.Button` 
+- `T:System.Windows.Forms.DataGridViewCheckBoxCell` control supports the [Toggle Pattern](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.togglepattern?view=netframework-4.7)
+ 
+__Improvements to PropertyBrowser control__
+
+The .NET Framework 4.7.1 adds the following improvements to the PropertyBrowser control: 
+- The __Details__ button in the error dialog that is displayed when the user enters an incorrect value in the T:System.Windows.Forms.PropertyGrid control supports the Expand/Collapse pattern, state and name change notifications, and a ControlType property with a value of "Button".
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: Can "ControlType property with a value of 'Button'" be expressed as an xref to ControlType.Field?

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+__How to opt in or out of these changes__
+  
+In order for the application to benefit from these changes, it must run on the .NET Framework 4.7.1 or later. The application can benefit from these changes in either of the following ways:
+- It is recompiled to target the .NET Framework 4.7.1.
+- It opts out of the legacy accessibility behaviors by adding the following [AppContext Switch](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the ```<runtime>``` section of the app config file and setting it to false, as the following example shows.
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: URLs should not include "en-us" -- they'll fail to localize in that case. That's true for many of the URLs here.

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+For an overview of UI automation, see the [UI Automation Overview](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-overview).
+
+__Added support for UI Automation patterns and properties__
+
+The .NET Framework 4.7.1 adds support for the fillowing UI automation patterns and properties:
+- The `T:System.Windows.Forms.ToolStripSplitButton` and `T:System.Windows.Forms.ComboBox` controls support the [Expand/Collapse pattern](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern)
+- The `T:System.Windows.Forms.ToolStripMenuItem` control supports the [ControlType](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-support-for-the-menubar-control-type) property value "MenuItem"
+- The `T:System.Windows.Forms.ToolStripItem` controls supports the [Name](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.automationelement.nameproperty?view=netframework-4.7) property and [Expand/Collapse pattern](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern)
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: URLs should not include "?view=netframework-4.7" Again, that's true of numerous URLs.

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+- `T:System.Windows.Forms.DataGridViewCheckBoxCell` control supports the [Toggle Pattern](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.togglepattern?view=netframework-4.7)
+ 
+__Improvements to PropertyBrowser control__
+
+The .NET Framework 4.7.1 adds the following improvements to the PropertyBrowser control: 
+- The __Details__ button in the error dialog that is displayed when the user enters an incorrect value in the T:System.Windows.Forms.PropertyGrid control supports the Expand/Collapse pattern, state and name change notifications, and a ControlType property with a value of "Button".
+- The [AccessibleRole](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.accessiblerole?view=netframework-4.7) of rows in PropertyGrid control have changed from "Row" to "Cell". The cell maps to UIA ControlType "DataItem", which allows it to support appropriate keyboard shortcuts and Narrator announcements.
+- Improved keyboard navigation between the grid and the ToolBar above it. Pressing "Shift-Tab" now selects the first ToolBar button, instead of the whole ToolBar
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: Toolbar should either be lowercased ("toolbar") or made a link (`T:System.Windows.Forms.ToolBar`)

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+- `T:System.Windows.Forms.Button`, `T:System.Windows.Forms.CheckBox`, `T:System.Windows.Forms.RadioButton`, `T:System.Windows.Forms.Label`, `T:System.Windows.Forms.LinkLabel` and `T:System.Windows.Forms.GroupBox` with`P:System.Windows.Forms.Control.Enabled` set to _false_, used a shaded color to rendered text in High Contrast themes, resulting in low contrast against the background. Now these controls use "Disabled Text" color defined by the OS. This fix applies to control with `P:System.Windows.Forms.Control.FlatStyle` set value other than `F:System.Windows.Forms.FlatStyle.System`. The latter controls are rendered by the OS.
+
+NOTE: Windows10 has changed values for some high contrast system colors. Windows Forms Framework is based on Win32 framework. For the best experience, run on the latest version of Windows and opt in to the latest OS changes by adding an app.manifest file in a test application and uncommenting the following code:
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: Win32 framework --> the Win32 framework

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+```
+    <!-- Windows 10 -->
+    <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
+```
+__Improved keyboard navigation__
+
+- When `T:System.Windows.Forms.ComboBox` control has `P:System.Windows.Forms.ComboBox.DropDownStyle` set to `F:System.Windows.Forms.DropDownStyle.DropDownList`, and is the first control in the TabOrder on the form, it is now displaying focus rectangle when the parent form is opened using the keyboard. Before this change, keyboard focus was on this control, but focus indicator was not rendered.
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: When --> When a
no comma after DropDownList
TabOrder --> tab order
it is now displaying -->it now displays a
but focus indicator --> but a focus indicator

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
+## 195: Accessibility improvements in Windows Forms controls
```
Comment (conniey) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: Are we going to abandon/close #477 then?

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @merriemcgaw. I think this looks ready to merge.  Should we merge this and close #477?

File: Documentation/compatibility/winforms-accessibility-changes-471.md (hunk 7 above)
Comment (merriemcgaw) [reply Documentation/compatibility/winforms-accessibility-changes-471.md]: Yep. I believe that is the plan.

**Comment by merriemcgaw** [comment]
Thanks @rpetrusha. I'll merge this and keep plugging away at my next group of bugs.

File: Documentation/compatibility/winforms-accessibility-changes-471.md (hunk 7 above)
Comment (Tanya-Solyanik) [reply Documentation/compatibility/winforms-accessibility-changes-471.md]: done

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+      </runtime>
+    </configuration>
+```
+Applications that target the .NET Framework 4.7.1 or later and want to preserve the legacy accessibility behavior can opt in to the use of legacy accessibility features by explicitly setting this AppContext switch to ```true```.
+
+For an overview of UI automation, see the [UI Automation Overview](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-overview).
+
+__Added support for UI Automation patterns and properties__
+
+The .NET Framework 4.7.1 adds support for the fillowing UI automation patterns and properties:
```
Comment (Tanya-Solyanik) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: following

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+- The `T:System.Windows.Forms.ToolStripDropDownButton` control has a [ControlType](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-support-for-the-menubar-control-type) property value of <xref:System.Windows.Automation.ControlType.MenuItem?displayProperty=fullName>
+- The `T:System.Windows.Forms.DataGridViewCheckBoxCell` control supports the [Toggle Pattern](https://docs.microsoft.com/dotnet/api/system.windows.automation.togglepattern)
+
+__Improvements to PropertyBrowser control__
+
+The .NET Framework 4.7.1 adds the following improvements to the PropertyBrowser control: 
+- The __Details__ button in the error dialog that is displayed when the user enters an incorrect value in the T:System.Windows.Forms.PropertyGrid control supports the Expand/Collapse pattern, state and name change notifications, and a [ControlType](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-support-for-the-menubar-control-type) property with a value of <xref:System.Windows.Automation.ControlType.MenuItem?displayProperty=fullName>.
```
Comment (Tanya-Solyanik) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: backticks are missing around T:System.Windows.Forms.PropertyGrid

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+- The `T:System.Windows.Forms.ToolStripMenuItem` control has a [ControlType](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-support-for-the-menubar-control-type) property value <xref:System.Windows.Automation.ControlType.MenuItem?displayProperty=fullName>
+- The `T:System.Windows.Forms.ToolStripItem` control supports the [Name](https://docs.microsoft.com/dotnet/api/system.windows.automation.automationelement.nameproperty) property and [Expand/Collapse pattern](https://docs.microsoft.com/dotnet/framework/ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern)
+- The `T:System.Windows.Forms.ToolStripDropDownItem` control supports [AccessibleEvents](https://docs.microsoft.com/dotnet/api/system.windows.forms.accessibleevents) indicating StateChange and NameChange when drop down is expanded or collapsed.
+- The `T:System.Windows.Forms.ToolStripDropDownButton` control has a [ControlType](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-support-for-the-menubar-control-type) property value of <xref:System.Windows.Automation.ControlType.MenuItem?displayProperty=fullName>
```
Comment (Tanya-Solyanik) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: this enum value is not displayed right:
<xref:System.Windows.Automation.ControlType.MenuItem?displayProperty=fullName> 

 It should be 
`F:System.Windows.Automation.ControlType.MenuItem`

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+
+__Use of OS-defined colors in High Contrast themes__
+
+- `T:System.Windows.Forms.Button` and `T:System.Windows.Forms.CheckBox` controls with `P:System.Windows.Forms.Control.FlatStyle` set to `F:System.Windows.Forms.FlatStyle.Standard`, which is the default style, now use OS-defined colors in High Contrast theme when selected. Previously, test and background colors were not contrasting and were hard to read.
+- `T:System.Windows.Forms.Button`, `T:System.Windows.Forms.CheckBox`, `T:System.Windows.Forms.RadioButton`, `T:System.Windows.Forms.Label`, `T:System.Windows.Forms.LinkLabel` and `T:System.Windows.Forms.GroupBox` with`P:System.Windows.Forms.Control.Enabled` set to _false_, used a shaded color to rendered text in High Contrast themes, resulting in low contrast against the background. Now these controls use "Disabled Text" color defined by the OS. This fix applies to control with `P:System.Windows.Forms.Control.FlatStyle` set value other than `F:System.Windows.Forms.FlatStyle.System`. The latter controls are rendered by the OS.
```
Comment (Tanya-Solyanik) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: This fix applies to control with `P:System.Windows.Forms.Control.FlatStyle` set value other than `F:System.Windows.Forms.FlatStyle.System`.

 I think this is mine, I was changing this sentence back and forth.. I meant this:

This fix applies to controls with `P:System.Windows.Forms.Control.FlatStyle` property set to a value other than `F:System.Windows.Forms.FlatStyle.System`.

File: Documentation/compatibility/winforms-accessibility-changes-471.md (hunk 10 above)
Comment (merriemcgaw) [reply Documentation/compatibility/winforms-accessibility-changes-471.md]: @rpetrusha, you had asked me to use the <xref: block for this item in the latest round of feedback. Can you give me more details on how to use them and what they should look like?

**Pull request merged**